src/lm/arpa-file-parser-test.cc

```diff
@@ -220,18 +220,18 @@ ngram 2=2\n\
 ngram 3=2\n\
 \n\
 \\1-grams: \n\
--5.2    a -3.3\n\
--3.4    \xCE\xB2\n\
-0.0     <s> -2.5\n\
--4.3    </s>\n\
+-5.2	a -3.3\n\
```

Comment:
The original version uses tab instead of \t or space. Do you want me to change it to \t?